### DISCUSSION TITLE
[V4] Help on setting it up with Symfony + Vite, please!

### DISCUSSION THREAD

**Discussion by 0neda** [tailwindlabs/tailwindcss#15114] [Help]
Hey guys. I've been trying to set-up Symfony & Vite + Tailwind V4 Beta1 but no success. For some reason, seems like the css isnt being generated as its not applying any classes, not really sure why.

I set Symfony & Vite following the guide on here: https://symfony-vite.pentatrion.com
And then tried setting up Tailwind with the V4 guide, but no success as I said above.

Any insights? Let me know if y'all need any files!

**Comment by didiramzi** [comment]
i'm facing the same issue, have you resolve it ?

**Reply by 0neda** [reply]
Hey @didiramzi! Sorry for the late response.

I cant remember if it was solved, but, on **new projects** I can use Symfony-Vite from Pentatrion just fine.
But, it only starts updating the page itself after importing Tailwind in the actual "app.css" file. If you have any problems, just follow the guide from https://symfony-vite.pentatrion.com/guide/installation.html

By the way, it seems easier if you **start by creating the project following the guide**, just so you know you haven't missed any steps or anything.

Hope it helped a bit, if not, let us know what is happening so we can try helping a bit more!
